### PR TITLE
Fixed warning in Pixel unit tests

### PR DESCRIPTION
**Pull request by jessemapel**
The Pixel unit tests were generating a warning.

## Description

There was a test using a number that doesn't fit in a double.

## Related Issue

#613 

## Motivation and Context

Removes build warnings

## How Has This Been Tested?

Pixel unit test has been re-built and passes with the change.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/PixelTests.cpp [hunk 1]
```diff
@@ -465,7 +465,6 @@ TEST(Pixel, static_ToString) {
   EXPECT_EQ(std::string("Lis"), Pixel::ToString(Isis::Lis));
   EXPECT_EQ(std::string("Lrs"), Pixel::ToString(Isis::Lrs));
   EXPECT_EQ(std::string("Null"), Pixel::ToString(Isis::Null));
-  EXPECT_EQ(std::string("Invalid"), Pixel::ToString(-1.0e+1000));
```
Comment (kberryUSGS) [inline isis/tests/PixelTests.cpp]: Is there any way we can test "Invalid" still without exceeding the valid range for doubles? Or is that what "Invalid" checks for?

Comment (krlberry) [inline isis/tests/PixelTests.cpp:468]: Is there any way we can test "Invalid" still without exceeding the valid range for doubles? Or is that what "Invalid" checks for?

Comment (jessemapel) [reply isis/tests/PixelTests.cpp:468]: This is checking for values below valid minimum. The problem is, all of the double values below valid minimum are special pixels or NaN.

The only way this happens is a double overflow, which I think is such an extreme outlier it's not worth testing.

Comment (krlberry) [reply isis/tests/PixelTests.cpp:468]: :+1: Sounds good to me, then!

**Review by krlberry — APPROVED** [review]
:+1:

**Pull request merged**